packages/labs/cli/test-project/src/element-a.ts

```diff
@@ -7,9 +7,13 @@
 import {LitElement, html, css} from 'lit';
 import {customElement, property} from 'lit/decorators.js';
 
+/**
```

Comment:
Let's make an issue for writing a series of test elements that might include:
* element that renders SR
* has properties
* fires events
* has slots
* kitchen sink

Reply:
https://github.com/lit/lit/issues/2854